packages/autorest.typescript/test/smoke/generated/purview-administration-rest/review/purview-administration-rest.api.md

```diff
@@ -557,11 +557,15 @@ interface ComplexReplacerConfigOutput {
     typeName?: string;
 }
 
+// Warning: (ae-forgotten-export) The symbol "PurviewMetadataPoliciesClientOptions" needs to be exported by the entry point index.d.ts
+//
 // @public
-function createClient(endpoint: string, credentials: KeyCredential, options?: ClientOptions): PurviewMetadataPoliciesClient;
+function createClient(endpoint: string, credentials: KeyCredential, { apiVersion, ...options }?: PurviewMetadataPoliciesClientOptions): PurviewMetadataPoliciesClient;
 
+// Warning: (ae-forgotten-export) The symbol "PurviewAccountClientOptions" needs to be exported by the entry point index.d.ts
```

Comment:
looks like when we generate the export here, we're only exporting `createClient`:
 ```
export { createClient } from "./purviewAccount";
```

Reply:
good catch!

Reply:
should have been resolved.